certify-sunbird-insurance.properties

```diff
@@ -9,25 +9,25 @@ mosip.certify.integration.audit-plugin=LoggerAuditService
 
 ## ------------------------------------------- Plugin specific usecase properties ------------------------------------------------------------
 mosip.certify.vciplugin.sunbird-rc.enable-psut-based-registry-search=false
-mosip.certify.vciplugin.sunbird-rc.issue-credential-url=http://demo-registry-credentials-service:3000/credentials-service/credentials/issue
+mosip.certify.vciplugin.sunbird-rc.issue-credential-url=http://demo-registry-credentials-service.sunbird-registry/credentials-service/credentials/issue
```

Comment:
why is this also named as .sunbird-registry and .sunbird like the rest of the url changes.

Reply:
registry is deployed in sunbird namespace
rest all services(identity, credential-schema, credential) are deployed in sunbird-registry namespace

Reply:
got it